src/popupbutton.cpp

```diff
@@ -45,7 +45,7 @@ void PopupButton::draw(NVGcontext* ctx) {
         NVGcolor textColor =
             mTextColor.w() == 0 ? mTheme->mTextColor : mTextColor;
 
-        nvgFontSize(ctx, (mFontSize < 0 ? mTheme->mButtonFontSize : mFontSize) * 1.5f);
+        nvgFontSize(ctx, (mFontSize < 0 ? mTheme->mButtonFontSize : mFontSize) * detail::__icon_scale);
```

Comment:
Fourth icon scale change.  This one used the original `ih *= 0.5` values, but it may need a different scale.